### DISCUSSION TITLE
Can Quarkus apps be run on AWS Gravitron v2?

### DISCUSSION THREAD

**Discussion by DGuhr** [quarkusio/quarkus#26090] [Q&A]
Hey :)

So, as the headline states: "Can Quarkus based applications run on AWS Gravitron v2?" Any experiences? General question for now, as we have this issue here: https://github.com/keycloak/keycloak/issues/11388 filed over at keycloak.

I am struggling a bit to get an AWS account short-term, and most likely this is due to a transient dependency from quarkus/ispn (Netty), so I thought why not ask you ppl first. 

Thanks in advance!

**Comment by DGuhr** [comment]
Ok, to specify this question a bit more and after further analysis: It seems that quarkus netty only has the dependency `io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.74.Final` bundled, but not the (needed) dependency

```
<dependency>
      <groupId>io.netty</groupId>
      <artifactId>netty-transport-native-epoll</artifactId>
      <classifier>linux-aarch_64</classifier>
      <version>4.1.74.Final</version>
    </dependency>
```
to make native transfers work on Aarch64. 

So, to rephrase: Is it possible to add this dependency, so native epoll and the respective `quarkus.vertx.prefer-native-transport=true` would work with Aarch64 / Graviton environments?

**Reply by gsmet** [reply]
I don't know, from what I can see, both are optional dependencies in our `quarkus-netty` extension. Are you sure we drag the Linux one automatically? If so, we need to find where.

/cc @cescoffier

**Reply by cescoffier** [reply]
As of today, Quarkus does not support native Netty transport in native mode.

**Reply by DGuhr** [reply]
@gsmet oh sorry 🙈  i re-checked, thought it was from quarkus-netty, but doing a `mvn dependency:tree -Dverbose` it seems to be loaded from our infinispan dependency directly instead of from netty.
